cms_form/tests/utils.py

```diff
@@ -32,6 +33,24 @@ def fake_request(form_data=None, query_string=None,
     return o_req
 
 
+def fake_session(env, **kw):
+    db = get_db_name()
+    env = api.Environment(env.cr, env.uid, {})
+    session = http.root.session_store.new()
+    session.db = db
+    session.uid = env.uid
+    session.login = 'admin'
```

Comment:
Are we sure `env.uid` above matches with admin ? What about using this with `sudo()` in a test ?